roles/ceph-facts/tasks/facts.yml

```diff
@@ -8,19 +8,13 @@
   set_fact:
     is_atomic: "{{ stat_ostree.stat.exists }}"
 
-- name: check if podman binary is present
-  stat:
-    path: /usr/bin/podman
-  register: podman_binary
+- name: import_tasks container_binary.yml
+  import_tasks: container_binary.yml
 
 - name: set_fact is_podman
   set_fact:
     is_podman: "{{ podman_binary.stat.exists }}"
```

Comment:
this should be move to container_binary file

Reply:
not needed